CHANGELOG.md

```diff
@@ -93,6 +93,7 @@ to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
     - math variable now allows rounding output with the ~ operator
     - French translation has been updated
     - `action` objective - cancels now the event, before other plugins check for it (better third-party support)
+- `NPC Glow` Glow NPC if spesific conditions met
```

Comment:
This belongs to line 37

Reply:
I just realized it on `changed` not `added` but what do you mean by belong line 37?

Reply:
i mean it should be inserted at line 37